python/core/qgsoptionalexpression.sip

```diff
@@ -13,6 +25,8 @@ class QgsOptionalExpression
 
 %TypeHeaderCode
 #include "qgsoptionalexpression.h"
+#include "qgsoptional.h"
+typedef QgsOptional<QgsExpression> QgsOptionalExpression;
```

Comment:
@m-kuhn I tried to use the method described [here](https://www.riverbankcomputing.com/pipermail/pyqt/2015-May/035893.html) but got this error
> python/core/qgsoptionalexpression.sip:29:36: error: typedef redefinition with different types ('QgsOptional<QgsExpression>' vs 'QgsOptionalExpression')
typedef QgsOptional<QgsExpression> QgsOptionalExpression;
                                   ^
../python/../src/core/qgsoptionalexpression.h:37:19: note: previous definition is here
class CORE_EXPORT QgsOptionalExpression : public QgsOptional<QgsExpression>
                  ^
1 error generated.



with or without the include of the main header `qgsoptionalexpression.h`

Do you have any idea?